tests/page-objects/reports/reports.po.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
   firstReport: () => element(by.css(`${reportListID} li:first-child`)),
   listLoader: () => element(by.css(`${reportListID} .loader`)),
   list: () => element(by.css(reportListID)),
-  reportByUUID: uuid => module.exports.list().all(by.css('li[data-record-id="' + uuid + '"]')),
+  reportByUUID: uuid => module.exports.list().all(by.css(`li[data-record-id="${uuid}"]`)).first(),
+  reportsByUUID: uuid => module.exports.list().all(by.css(`li[data-record-id="${uuid}"]`)),
```

Comment:
I see you got the string template stuff, but perhaps forgot about the request to extract to a variable. I was going to suggest this...
```
  const matchedListItems = by.css(`li[data-record-id="${uuid}"]`);
  reportByUUID: uuid => module.exports.list().all(matchedListItems).first(),
  reportsByUUID: uuid => module.exports.list().all(matchedListItems),
```
...but then realized this is all inline. I'm tempted to say "eh, nevermind, let's leave it for now" but this is still pretty quick to make easier on the reader. Similar to how lines 15 and 16 work, please just make functions for `reportByUUID` and reportsByUUID with the logic in them (not all inline here). Then, a nice benefit is that the implementation of `reportByUUID` will just be to call `reportsByUUID` and then call `first` on the result.